cypress/cypress/integration/record.spec.ts

```diff
@@ -2078,6 +2080,96 @@ describe("Record", () => {
     cy.get("[data-cy=undo-question-btn]").should("be.disabled");
   });
 
+  it("Verify that transcript markdown loads as rich text: Q1", () => {
+    cyMockDefault(cy, {
+      mentor: clintMarkdown,
+      questions: chatQuestions,
+      gqlQueries: [
+        mockGQL("UploadTaskDelete", { me: { uploadTaskDelete: true } }),
+        mockGQL("UpdateAnswer", { me: { updateAnswer: true } }),
+        mockGQL("UpdateQuestion", { me: { updateQuestion: true } }),
+        mockGQL("ImportTask", { importTask: null }),
+        mockGQL("FetchUploadTasks", [{ me: { uploadTasks: [] } }]),
+      ],
+    });
+    cy.visit("/record?videoId=A1_1_1");
+    cy.get("[data-text]").should("be.visible");
+  });
+
+  it("Verify that transcript markdown loads as rich text: Q2", () => {
```

Comment:
You don't need to create a whole new test for a separate question, just have cypress click next on the record page if they are part of the same set of questions, or run another cy.visit in the same test